src/views/Article.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div class="view">
+        <div class="toolbar">
+            <div @click="onBack">Back</div>
+            <div @click="onConfirm">Confirm</div>
+        </div>
+        <div class="article" @click="onClick" v-html="currentArticle"></div>
+    </div>
+</template>
+<script>
+    import { mapGetters, mapActions } from 'vuex'
+    export default {
+      name: 'Article',
+      props: ['article'],
+      computed: mapGetters(['currentArticle']),
+      methods: {
+        ...mapActions(['fetchArticle', 'setText', 'setImg']),
+        onBack: function() {
+          this.$router.go(-1)
+        },
+        onConfirm: function() {
+          const selectedText = document.querySelector('.selected-text')
+          const text = selectedText && selectedText.innerText
+
+          const selectedImage = document.querySelector('.selected-image')
+          const imageSrc = selectedImage && selectedImage.src
+
+            if (text && imageSrc) {
+              this.setText(text)
+              this.setImg(imageSrc)
+              this.$router.push( { name: 'Story' } )
+            }
+        },
+        onClick: function(e) {
+          e.preventDefault()
+          if (e.target.tagName === 'IMG') {
+            const selectedImage = document.querySelector('.selected-image')
+            if (selectedImage) {
+              selectedImage.classList.remove('selected-image')
+              if (selectedImage !== e.target) {
+                e.target.classList.add('selected-image')
+              }
+            } else {
+              e.target.classList.add('selected-image')
+            }
+          } else {
+            const selectedText = document.querySelector( '.selected-text' )
+            if ( selectedText ) {
+              selectedText.classList.remove( 'selected-text' )
+              if (selectedText !== e.target) {
+                e.target.classList.add('selected-text')
+              }
+            } else {
+              e.target.classList.add( 'selected-text' )
```

Comment:
I can select the <figure /> as the text
<img src="https://user-images.githubusercontent.com/2560096/135118294-12070095-8ee9-4d20-840c-f7ce041a036c.png" width="500"/>